src/Codeception/Lib/Connector/Yii2.php

```diff
@@ -275,6 +275,16 @@ protected function persistDb()
 
     private function mockAssetManager()
     {
-        $this->app->set('assetManager', Stub::make('yii\web\AssetManager', ['bundles' => false]));
```

Comment:
By default is `'basePath' => '@webroot/assets'`. That why you should set correct `@webroot` alias [like in real application](http://www.yiiframework.com/doc-2.0/guide-concept-aliases.html#predefined-aliases). 
> @webroot, the Web root directory of the currently running Web application. **It is determined based on the directory containing the entry script**